Integrations API doc refactoring (#4537)
DataDog/documentation: content/en/api/integrations_gcp/gcp_create.md

---
title: Create a GCP integration
type: apicontent
order: 17.2
external_redirect: /api/#create-a-gcp-integration
---

## Create a GCP integration

Create a Datadog-Google Cloud Platform integration.

**Note**:

> * Using the `POST` method updates your integration configuration by **adding** your new configuration to the existing one in your Datadog organization.

Reviewer: I'm a bit unclear here - does this add an additional config file, or does it combine your existing config with the new one?

Author: Yup this add a new configuration file, the `PUT` method would combine the config pushed with the existing configuration if any.

* Using the `PUT` method updates your integration configuration by **replacing** your current configuration with the new one sent to your Datadog organization.

##### ARGUMENTS

All of the following fields' values are provided by the JSON service account key file created in the [GCP Console for service accounts][1]; Refer to the [Datadog-Google Cloud Platform integration installation instructions][2] to see how to generate one for your organization.
For further references, consult the [Google Cloud service account documentation][3].

* **`type`** [*required*]:

>     Should be `service_account`, it can be found in your JSON service account key.

Reviewer: ```suggestion
    The value for `service_account` found in your JSON service account key.
```
